it/config/DocumentationControllerISpec.scala

```diff
@@ -90,11 +85,12 @@ class DocumentationControllerISpec extends IntegrationBaseSpec {
   }
 
   "an OAS documentation request" must {
-    List(Version1, Version2, Version3, Version4).foreach { version =>
+    //v4 hidden by MTDSA-20447
+    List(Version1, Version2, Version3 /*Version4*/).foreach { version =>
```

Comment:
Here you've commented out "Version 4" but in the previous file you removed it. Should we remove it here too just to make it all consistent?